packages/cdk/lib/cloudquery/index.ts

```diff
@@ -168,11 +169,48 @@ export function addCloudqueryEcsCluster(
 			name: 'AwsCostExplorer',
 			description:
 				'Collecting Cost Explorer information for the Worflow account. This requires the use of paid AWS APIs so we are trialling it in a single account first',
```

Comment:
I think we'll want to update this description; perhaps something like:

```suggestion
				'Collects daily AWS costs (aggregated by App, Stack and Stage tags). We aim to keep historical data for this table (unlike other tables)',
```